src/jit/importer.cpp

```diff
@@ -3607,7 +3608,33 @@ GenTreePtr Compiler::impIntrinsic(CORINFO_CLASS_HANDLE  clsHnd,
             retNode = op1;
             break;
 #endif
-
+        // Implement ByReference Ctor.  This wraps the assignment of the ref to the span field.
```

Comment:
If I understand correctly, this is an abstraction that is currently being used only for the span case, but could apply to other types with a ByRef-like fields as well. If so, it might be better to describe this as "This wraps the assignment of the ref to a byref-like field" or something like that.